io.github.maurycyliebner.enve.metainfo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should be sent upstream if possible

Reply:
I know: :wink:
https://github.com/MaurycyLiebner/enve/pull/114